include/model/CAnnotatedProbabilityBuilder.h

```diff
@@ -47,46 +47,34 @@ class MODEL_EXPORT CAnnotatedProbabilityBuilder : private core::CNonCopyable {
 
     CAnnotatedProbabilityBuilder(SAnnotatedProbability& annotatedProbability,
                                  std::size_t numberAttributeProbabilities,
-                                 function_t::EFunction function,
-                                 std::size_t numberOfPeople);
+                                 function_t::EFunction function);
 
     void attributeProbabilityPrior(const maths::common::CMultinomialConjugate* prior);
     void personAttributeProbabilityPrior(const maths::common::CMultinomialConjugate* prior);
-    void personFrequency(double frequency, bool everSeenBefore);
     void probability(double p);
     void anomalyScoreExplanation(SAnnotatedProbability::TAnomalyScoreExplanation explanations);
+    SAnnotatedProbability::TAnomalyScoreExplanation& anomalyScoreExplanation();
```

Comment:
I think now you have a return by writeable reference I would remove `void anomalyScoreExplanation(SAnnotatedProbability::TAnomalyScoreExplanation explanations);`: you should be able to replace it with `builder.anomalyScoreExplanation() = ...`

Reply:
changed